### PR TITLE
Hardening/r4 apib fixes

### PR DESCRIPTION
**Pull request by fgalan**
Landing recent improvments in orioncontextbroker.apib for FIWARE R4 into develop branch, thanks to @FiwareULPGC contributions!

### CODE REVIEW

File: doc/apiary/orioncontextbroker.apib [hunk 1]
```diff
@@ -224,9 +223,9 @@ And the response will be;
 </center>
 + Request (application/json)
 
-    + Headers (application/json)
+    + Headers 
     
-        Accept: application/json
+                Accept: application/json
```
Comment (kzangeli) [inline doc/apiary/orioncontextbroker.apib]: Too much indentation?

File: doc/apiary/orioncontextbroker.apib [hunk 2]
```diff
@@ -3037,145 +3043,147 @@ The result of the query would be Madrid, Leganes and Alcobendas.As shown in the
...
+                            "latitude": "6",
+                            "longitude": "6"
+                            },
+                            {
+                            "latitude": "6",
+                            "longitude": "0"
+                            }
+                        ]
+                        }
                         }
```
Comment (kzangeli) [inline doc/apiary/orioncontextbroker.apib]: Bad indentation in all the 'scopes' block ...

File: doc/apiary/orioncontextbroker.apib [hunk 3]
```diff
@@ -3037,145 +3043,147 @@ The result of the query would be Madrid, Leganes and Alcobendas.As shown in the
...
+                "restriction": {
+                    "scopes": [
+                        {
+                        "type" : "FIWARE::Location",
+                        "value" : {
+                        "circle": {
+                        "centerLatitude": "40.418889",
+                        "centerLongitude": "-3.691944",
+                        "radius": "15000"
+                        }
```
Comment (kzangeli) [inline doc/apiary/orioncontextbroker.apib]: Indentation ...

File: doc/apiary/orioncontextbroker.apib [hunk 4]
```diff
@@ -3440,20 +3449,20 @@ You may wonder why accumulator-server.py is getting this message if you don't ac
...
 + Response 200 (application/json)
 
-        {
-            "subscribeResponse": {
-            "duration": "P1M", 
-            "subscriptionId": "552f7481983d79a38fbd5a74"
+            {
+                "subscribeResponse": {
+                "duration": "P1M", 
+                "subscriptionId": "552f7481983d79a38fbd5a74"
```
Comment (kzangeli) [inline doc/apiary/orioncontextbroker.apib]: Indentation for duration and subscriptionId

File: doc/apiary/orioncontextbroker.apib (hunk 1 above)
Comment (fgalan) [inline doc/apiary/orioncontextbroker.apib]: It seems that the FABRE tool that process the .apib file needs such indentation.

NTC

File: doc/apiary/orioncontextbroker.apib (hunk 2 above)
Comment (fgalan) [inline doc/apiary/orioncontextbroker.apib]: Same here. NTC.

File: doc/apiary/orioncontextbroker.apib (hunk 3 above)
Comment (fgalan) [inline doc/apiary/orioncontextbroker.apib]: Same here. NTC.

File: doc/apiary/orioncontextbroker.apib (hunk 4 above)
Comment (kzangeli) [inline doc/apiary/orioncontextbroker.apib]: Same here, I guess ... :-)

NTC

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**